### ISSUE TITLE
Additional windows with workspace_layout tabbed spawned in a vertical split

### ISSUE THREAD

**Issue by thejan2009** [i3/i3#2402]
Output of `i3 --moreversion 2>&- || i3 --version`:

Binary i3 version:  4.12 (2016-03-06, branch "4.12") © 2009 Michael Stapelberg and contributors
Running i3 version: 4.12 (2016-03-06, branch "4.12") (pid 13073)
Loaded i3 config: /home/js/.i3/config (Last modified: Sun 24 Jul 2016 11:38:29 AM CEST, 4916 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: i3

URL to a logfile as per http://i3wm.org/docs/debugging.html:

http://logs.i3wm.org/logs/5743314154291200.bz2

**What I did:**
Switch to an empty workspace, create 2 windows (terminals in this case), focus parent, create a new window.

**What I saw:**
The last window was created in a vertical split layout, although there is workspace_layout tabbed in config.

**What I expected instead:**
New window created in a new tabbed container (probably T[T[urxvtc urxvtc] urxvtc], but got V[T[urxvtc urxvtc] urxvtc] instead)?

**Comment by Airblader** [comment]
I haven't reproduced this, but it does sound like a bug to me. On a side note, I'd personally wish we didn't have this "workspace layout" concept whatsoever. But we do, so we gotta fix it.

**Comment by thejan2009** [comment]
I could reproduce this "bug" even with only one container - workspace_layout tabbed, create window, focus parent (at this point workspace is probably the focused container), create new window. It seems that direct children of workspaces are created in a vertical split, but they do inherit the layout from config, so without focusing the parent, everything works as expected.

**Comment by thejan2009** [comment]
Bump.

**Comment by Airblader** [comment]
Reading it again I'm actually starting to think this isn't a bug but works as designed. But again, I'll let @stapelberg comment on the underlying idea.

**Comment by stapelberg** [comment]
Thanks for your patience with my reply on this issue.

First, to clarify the intention: with `workspace_layout tabbed`, any new window that is opened at the workspace level (i.e. after you use `focus parent` in your example) should be opened in a `tabbed` split container.

Hence, the expected outcome is `W[T[term term] T[term]]` (W stands for workspace here).

Now, looking at your logfile, that is actually what happens:
```
07/24/2016 12:59:33 PM - con.c:con_new_skeleton:52 - opening window
07/24/2016 12:59:33 PM - con.c:_con_attach:135 - Parent is a workspace. Applying default layout...
07/24/2016 12:59:33 PM - workspace.c:workspace_attach_to:852 - Attaching a window to workspace 0x16cf3b0 / 7:g
07/24/2016 12:59:33 PM - workspace.c:workspace_attach_to:859 - Non-default layout, creating a new split container
07/24/2016 12:59:33 PM - con.c:con_new_skeleton:52 - opening window
07/24/2016 12:59:33 PM - x.c:x_con_init:150 - Adding window 0x0060022a to lists
07/24/2016 12:59:33 PM - x.c:x_con_init:154 - adding new state for window id 0x0060022a
07/24/2016 12:59:33 PM - workspace.c:workspace_attach_to:868 - Attaching new split 0x1757e70 to workspace 0x16cf3b0
[…]
07/24/2016 12:59:33 PM - manage.c:manage_window:326 - new container = 0x1757ca0
[…]
07/24/2016 12:59:33 PM - render.c:render_con:51 - Rendering node 0x1757e70 / (null) / layout 2 / children 1
07/24/2016 12:59:33 PM - render.c:render_con:156 - child at (2326, 19) with (960 x 1061)
07/24/2016 12:59:33 PM - render.c:render_con:51 - Rendering node 0x1757ca0 / (null) / layout 6 / children 0
[…]
```

In fact, this behavior is verified by testcase  https://github.com/i3/i3/blob/6b89690a3f923fbd5b7b3988ed7a8934af6038df/testcases/t/167-workspace_layout.t#L54-L86

In conclusion: I’m not quite sure what the bug is — the code seems correct, and the logfile indicates everything seems fine.

@thejan2009, could you elaborate please on what the issue is? If you haven’t seen it, possibly watching https://www.youtube.com/watch?v=AWA8Pl57UBY makes it easier to talk about expectation and reality in terms of the container tree structure.

**Comment by thejan2009** [comment]
Thanks for the reply. I probably should have been more precise with my questions.

To rephrase your explanation - workspace is always vertically or horizontally split, while the containers that are its direct descendants inherit the layout from ```workspace_layout``` setting. 

Is that right? If it is, there is no issue. I was simply asking for clarification to better understand the reasons behind this behaviour - in [sway](https://github.com/SirCmpwn/sway) (i3-like wayland compositor), ```workspace_layout``` works differently and I plan to make it behave just like i3. The issue was raised to see if you overlooked something - and you didn't :).

By the way, thanks for making this awesome piece of software - I've been using it for the last 4 years.

**Comment by stapelberg** [comment]
> To rephrase your explanation - workspace is always vertically or horizontally split, while the containers that are its direct descendants inherit the layout from workspace_layout setting.

Correct.

> By the way, thanks for making this awesome piece of software - I've been using it for the last 4 years.

I’m glad you like it :).

Closing the issue as we’ve clarified the behavior.
